docs/protocol/specifications/data.md

```diff
@@ -427,12 +437,8 @@ The transaction signature hash is the [SHA3-256](#sha3) of the following structu
 Field                   | Type                                      | Description
 ------------------------|-------------------------------------------|----------------------------------------------------------
 Transaction ID          | sha3-256                                  | Current [transaction ID](#transaction-id).
-Input Index             | varint31                                  | Index of the current input encoded as [varint31](#varint31).
-Output Commitment Hash  | sha3-256                                  | [SHA3-256](#sha3) of the output commitment from the output being spent by the current input. Issuance input uses a hash of an empty string.
-
-Note 1. Including the spent output commitment makes it easier to verify the asset ID and amount at signing time, although those values are already committed to via the input's [outpoint](#outpoint).
+Input Index             | varint31                                  | Index of a given input encoded as [varint31](#varint31).
```

Comment:
Will keep `Input Index` for now, we'll have "Input ID" meaning the hash-based ID of an entry in the txgraph redesign.